web/crux-ui/src/models/grpc/google/protobuf/timestamp.ts

```diff
@@ -216,7 +208,3 @@ if (_m0.util.Long !== Long) {
   _m0.util.Long = Long as any
   _m0.configure()
 }
-
-function isSet(value: any): boolean {
-  return value !== null && value !== undefined
-}
```

Comment:
Why always updating this file? (I know it's autogenerated)

Reply:
I've also seen this, I think there might be some generator mismatch? Because I've seen older versions of this file in other PRs, but I'm not sure :/